pkg/component/blackboxexporter/blackboxexporter_test.go

```diff
@@ -82,7 +82,6 @@ var _ = Describe("BlackboxExporter", func() {
 
 	Describe("#Deploy", func() {
 		var (
-			vpaEnabled    = false
```

Comment:
Why did you remove this from the test?

Reply:
It is not required, the length is tested inside the Contexts for `w/` and `w/o` vpa.